File paths not resolving properly on Windows

Using Windows 7, node 64bit v.6.11.1

It's not receiving file paths correctly.

Gets error message in terminal when running npm start:
 
```
Project is running at http://localhost:9000/
webpack output is served from /
Content not from webpack is served from C:\Users\giggs\Documents\robpack\demo\dist
webpack: wait until bundle finished: /
Hash: 53cdf8733c669c76be58
Version: webpack 3.3.0
Time: 4749ms
                                                          Asset     Size  Chunks
          Chunk Names
                                                      bundle.js   355 kB       0  [emitted
]  [big]  main
                                                     index.html     2 kB          [emitted
]
bower_components/webcomponentsjs/custom-elements-es5-adapter.js  1.62 kB          [emitted
]
        bower_components/webcomponentsjs/webcomponents-hi-ce.js  23.6 kB          [emitted
]
                   bower_components/webcomponentsjs/gulpfile.js  5.09 kB          [emitted
]
     bower_components/webcomponentsjs/webcomponents-hi-sd-ce.js  72.6 kB          [emitted
]
           bower_components/webcomponentsjs/webcomponents-hi.js  9.72 kB          [emitted
]
       bower_components/webcomponentsjs/webcomponents-loader.js  3.15 kB          [emitted
]
         bower_components/webcomponentsjs/webcomponents-lite.js  83.5 kB          [emitted
]
        bower_components/webcomponentsjs/webcomponents-sd-ce.js  64.3 kB          [emitted
]
  [39] multi (webpack)-dev-server/client?http://localhost:9000 ./src/index.js 40 bytes {0}
 [built]
  [40] (webpack)-dev-server/client?http://localhost:9000 5.59 kB {0} [built]
  [41] ./node_modules/url/url.js 23.3 kB {0} [built]
  [43] ./node_modules/url/util.js 314 bytes {0} [built]
  [47] ./node_modules/strip-ansi/index.js 161 bytes {0} [built]
  [48] ./node_modules/ansi-regex/index.js 135 bytes {0} [built]
  [49] (webpack)-dev-server/client/socket.js 856 bytes {0} [built]
  [50] ./node_modules/sockjs-client/lib/entry.js 244 bytes {0} [built]
  [81] (webpack)-dev-server/client/overlay.js 3.6 kB {0} [built]
  [86] (webpack)/hot/emitter.js 77 bytes {0} [built]
  [87] ./node_modules/events/events.js 8.33 kB {0} [built]
  [88] ./src/index.js 44 bytes {0} [built]
  [89] ./src/my-element.html 2.78 kB {0} [built]
  [90] ./node_modules/date-fns/format/index.js 10.1 kB {0} [built]
 [103] ./node_modules/polymer-webpack-loader/register-html-template.js 1.92 kB {0} [built]

    + 89 hidden modules

ERROR in ./src/my-element.html
obpackdemower_componentspolymerpolymer-element.html' in 'C:\Users\giggs\Documents\robpack\
demo\src'
 @ ./src/my-element.html 5:0-88
 @ ./src/index.js
 @ multi (webpack)-dev-server/client?http://localhost:9000 ./src/index.js
Child html-webpack-plugin for "index.html":
       [0] ./node_modules/html-webpack-plugin/lib/loader.js!./src/index.ejs 2.49 kB {0} [b
uilt]
       [1] ./node_modules/lodash/lodash.js 540 kB {0} [built]
       [2] (webpack)/buildin/global.js 509 bytes {0} [built]
       [3] (webpack)/buildin/module.js 517 bytes {0} [built]
webpack: Failed to compile.
```
and in the console  

```
Uncaught Error: Cannot find module "C:UsersgiggsDocuments
obpackdemoower_componentspolymerpolymer-element.html"
    at webpackMissingModule (bundle.js:9646)
    at Object.<anonymous> (bundle.js:9646)
    at __webpack_require__ (bundle.js:20)
    at Object.<anonymous> (bundle.js:9635)
    at __webpack_require__ (bundle.js:20)
    at Object.<anonymous> (bundle.js:4097)
    at __webpack_require__ (bundle.js:20)
    at module.exports.ctor.super_ (bundle.js:63)
    at bundle.js:66
```

I'm seeing similar in a simple test that places a paper-button on the page. The below error is coming from webpack which has the backslashes missing in the path to the paper button

ERROR in ./bower_components/paper-button/paper-button.html
Module not found: Error: Can't resolve 'C:dev   estpolymerwebpacower_componentspolymerpolymer
.html' in 'C:\dev\test\polymerwebpack\bower_components\paper-button'
 @ ./bower_components/paper-button/paper-button.html 2:0-74
 @ ./app.js

I've uploaded the code to this repository for reference if helpful:

https://github.com/MarkJamesHoward/PolymerWebpackLoader.git

--- a/src/index.js
+++ b/src/index.js
@@ -10,7 +10,6 @@ import sourceMap from 'source-map';
 const domPred = predicates.AND(predicates.hasTagName('dom-module'));
 const linkPred = predicates.AND(predicates.hasTagName('link'));
 const scriptsPred = predicates.AND(predicates.hasTagName('script'));
-const windowsPathRegExp = new RegExp('\\\\', 'g');
 
 class ProcessHtml {
   constructor(content, loader) {
@@ -61,7 +60,7 @@ class ProcessHtml {
         const ignoredFromPartial = ignoreLinksFromPartialMatches.filter(partial => href.indexOf(partial) >= 0);
 
         if (ignoreLinks.indexOf(href) < 0 && ignoredFromPartial.length === 0) {
-          source += `\nimport '${path.replace(windowsPathRegExp, '\\\\')}';\n`;
+          source += `\nimport '${path.replace(/\\/g, '\\\\')}';\n`;
           lineCount += 2;
         }
       }
@@ -154,7 +153,7 @@ RegisterHtmlTemplate.toBody('${minimized.replace(/'/g, "\\'")}');
         const parseSrc = url.parse(src);
         if (!parseSrc.protocol || !parseSrc.slashes) {
           const path = osPath.join(osPath.dirname(this.currentFilePath), src);
-          source += `\nimport '${path.replace(windowsPathRegExp, '\\\\')}';\n`;
+          source += `\nimport '${path.replace(/\\/g, '\\\\')}';\n`;
           lineOffset += 2;
         }
       } else {

--- a/src/index.js
+++ b/src/index.js
@@ -10,6 +10,7 @@ import sourceMap from 'source-map';
 const domPred = predicates.AND(predicates.hasTagName('dom-module'));
 const linkPred = predicates.AND(predicates.hasTagName('link'));
 const scriptsPred = predicates.AND(predicates.hasTagName('script'));
+const windowsPathRegExp = new RegExp('\\\\', 'g');
 
 class ProcessHtml {
   constructor(content, loader) {
@@ -60,7 +61,7 @@ class ProcessHtml {
         const ignoredFromPartial = ignoreLinksFromPartialMatches.filter(partial => href.indexOf(partial) >= 0);
 
         if (ignoreLinks.indexOf(href) < 0 && ignoredFromPartial.length === 0) {
-          source += `\nimport '${path}';\n`;
+          source += `\nimport '${path.replace(windowsPathRegExp, '\\\\')}';\n`;
           lineCount += 2;
         }
       }
@@ -153,7 +154,7 @@ RegisterHtmlTemplate.toBody('${minimized.replace(/'/g, "\\'")}');
         const parseSrc = url.parse(src);
         if (!parseSrc.protocol || !parseSrc.slashes) {
           const path = osPath.join(osPath.dirname(this.currentFilePath), src);
-          source += `\nimport '${path}';\n`;
+          source += `\nimport '${path.replace(windowsPathRegExp, '\\\\')}';\n`;
           lineOffset += 2;
         }
       } else {

